core/src/main/groovy/noe/tomcat/configure/ConnectorAttributesTransformer.groovy

```diff
@@ -94,6 +94,10 @@ class ConnectorAttributesTransformer {
         res.put('SSLEnabled', connector.getSslEnabled())
       }
 
+      if (connector.getProtocol() != null && !connector.getProtocol().isEmpty()) {
```

Comment:
This should be handled by `CommonConnectorTransformer#transform()` (line 57). It doesn't working or what is the point of change please?

Reply:
Yop... you are right. I've just overlooked it.